### PR TITLE
Add option to hide mouse while typing

### LINKED ISSUE #819: Make configurable: hide mouse on keyboard input
This is currently already the case but it cannot be disabled.

So, make it configurable.

### PR DESCRIPTION
**Pull request by Yaraslaut**
Closes #819

### CODE REVIEW

File: src/contour/contour.yml [hunk 1]
```diff
@@ -220,6 +220,11 @@ profiles:
             # whether or not to hide the scrollbar when in alt-screen.
             hide_in_alt_screen: true
 
+        # mouse setting
+        mouse:
+            # whether or not to hide mouse when typing
```
Comment (christianparpart) [inline src/contour/contour.yml:225]:
```suggestion
            # whether or not to hide mouse when typing
            #
            # Default value: true
```
I think it's worth mentioning the default (or simply just to stay consistent with most of the options) :)

**Review by christianparpart — COMMENTED** [review]
I think such an option is worth adding to the changelog, as well. :)

**Comment by Yaraslaut** [comment]
> I think such an option is worth adding to the changelog, as well. :)

Done

**Review by christianparpart — APPROVED** [review]
Many thanks, @Yaraslaut :)

**Pull request merged**